aspnetBlazor/Properties/launchSettings.json

```diff
@@ -3,16 +3,25 @@
     "windowsAuthentication": false,
     "anonymousAuthentication": true,
     "iisExpress": {
-      "applicationUrl": "http://localhost:32394",
-      "sslPort": 44349
+      "applicationUrl": "http://localhost:19325",
+      "sslPort": 44386
     }
   },
   "profiles": {
     "__change__": {
       "commandName": "Project",
       "dotnetRunMessages": true,
       "launchBrowser": true,
-      "applicationUrl": "https://localhost:7001;http://localhost:5191",
+      "applicationUrl": "http://localhost:5270",

```

Comment:
Does this change the default port? Then, we should also change the port on `docker-compose.yml` and on `templates.json`. Also, there is a mention to the 7001 port on the `Program.cs`

Reply:
I do not think that this is really used for something on our workflow, this appears to be something related to Microsoft IIS. But I think that the best is change here the port.